### PR TITLE
Added ingressClassName parameter in ingress chart

### PR DESCRIPTION
**Pull request by sanhidalgoo**
IngressClassName is a new parameter in ingress chart.

### CODE REVIEW

**Comment by ameijer** [comment]
@sanhidalgoo thank you - can you increment the chart minor version please?

**Comment by ameijer** [comment]
also, can you add documentation to the chart README?

**Comment by sanhidalgoo** [comment]
@ameijer Of course! I just to add de docs and I changed the minor version.

File: charts/ingress/templates/ingress.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ metadata:
 {{- end }}
   name: {{ template "ingress.name" . }}
 spec:
+{{- if .Values.ingressClassName }}
+  ingressClassName: {{ .Values.ingressClassName }}
+{{- end -}}
```
Comment (ameijer) [inline charts/ingress/templates/ingress.yaml:20]: I don't think you want an - at the end of the end block. I think that is trimming your whitespace

**Review by ameijer — CHANGES_REQUESTED** [review]

File: charts/ingress/templates/ingress.yaml (hunk 1 above)
Comment (sanhidalgoo) [reply charts/ingress/templates/ingress.yaml]: Good! Thanks for the review! I just deleted the - at the end of the end block!

**Review by ameijer — APPROVED** [review]

**Pull request merged**